mesh/meshdb.go

```diff
@@ -678,3 +684,33 @@ func (m *MeshDB) ContextuallyValidBlock(layer types.LayerID) (map[types.BlockID]
 
 	return validBlks, nil
 }
+
+func (m *MeshDB) Persist(key []byte, v interface{}) error {
+	var buf bytes.Buffer
+	enc := gob.NewEncoder(&buf)
```

Comment:
I thought we're using xdr ? gob is a golang thing

Reply:
this is local data so its allowed, https://golang.org/pkg/encoding/gob/
changing encoders is fairly easy give it a look

Reply:
yes https://golang.org/pkg/encoding/gob/ what do you think ?

Reply:
I didn't read the whole thing, I'm familiar with gob. just thought that we should keep using the same serialization we've been using until now and not introduce new serialization formats into the project.